noobaa_sa/account.py

```diff
@@ -0,0 +1,179 @@
+"""
+Module which contain account operations like create, delete, list and update
+"""
+
+import logging
+
+from abc import ABC, abstractmethod
+from framework import config
+from framework.connection import SSHConnection
+from noobaa_sa.defaults import MANAGE_NSFS
+from noobaa_sa.exceptions import (
+    AccountCreationFailed,
+    AccountListFailed,
+    AccountDeletionFailed,
+)
+
+log = logging.getLogger(__name__)
+
+
+class Account(ABC):
+    """
+    A base class for Account related operations.
+    Should be inherited by specific deployment type like DB and NSFS
+    """
+
+    def __init__(self, account_json):
+        """
+        Initialize necessary variable
+
+        Args:
+            account_json (str): Path to account json file
+
+        """
+        self.account_json = account_json
+        self.manage_nsfs = MANAGE_NSFS
+        self.config_root = config.ENV_DATA["config_root"]
+        ssh_conn = SSHConnection()
+        self.conn = ssh_conn.get_connection()
```

Comment:
```suggestion
        self.conn = SSHConnection().connection
```

You can then use connection like this.

Reply:
done